art/preprocessing/standardisation_mean_std/standardisation_mean_std_tensorflow.py

```diff
@@ -40,7 +41,11 @@ class StandardisationMeanStdTensorFlowV2(PreprocessorTensorFlowV2):
     params = ["mean", "std", "apply_fit", "apply_predict"]
 
     def __init__(
-        self, mean: float = 0.0, std: float = 1.0, apply_fit: bool = True, apply_predict: bool = True,
+        self,
+        mean: Union[float, Sequence, np.ndarray] = 0.0,
+        std: Union[float, Sequence, np.ndarray] = 1.0,
```

Comment:
I agree with adding `np.ndarray` but is `Sequence` required?

Reply:
I removed `Sequence` type.